server/start.go

```diff
@@ -172,7 +172,7 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 			}
 
 			err = wrapCPUProfile(serverCtx, func() error {
-				return start(serverCtx, clientCtx, appCreator, withCMT, opts)
+				return start(cmd.Context(), serverCtx, clientCtx, appCreator, withCMT, opts)
```

Comment:
For my understanding why was this needed (asking because I don't see this change in https://github.com/dydxprotocol/cosmos-sdk/pull/30/files)

Reply:
It is to allow the caller to shutdown cosmos using the `context` that is associated with the command.

It was added later in https://github.com/dydxprotocol/cosmos-sdk/commit/b48250d4e0efacbd40f3e0b980c1e1004d29e351